online.php

```diff
@@ -159,7 +159,9 @@
 	{
 		case "sqlite":
 			$sessions = array();
-			$query = $db->simple_select("sessions", "sid", "time > {$timesearch}");
+			$query = $db->simple_select("sessions", "MAX(sid)", "time > {$timesearch}", array(
+			    'group_by' => 'uid, ip',
+            ));
```

Comment:
Indentation here is off - it seems to use spaces for the closing `));` rather than tabs.

```suggestion
			));
```